### PR TITLE
add download doc.

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Close https://github.com/neo-project/neo-express/issues/250.

### CODE REVIEW

**Review by Liaojinghui — APPROVED** [review]

File: docs/command-reference.md [hunk 1]
```diff
@@ -464,6 +464,31 @@ Arguments:
...
+  -i|--input <INPUT>    Path to neo-express data file
+  -h|--height <HEIGHT>  Block height to get contract state for
+                        Default value is: 0.
+  -f|--force[:<FORCE>]  Replace contract and storage if it already exists
+                        Defaults to None if option unspecified, All if option value unspecified
+                        Allowed values are: None, All, ContractOnly, StorageOnly.
+  -?|--help             Show help information.
+```
+
+Download contract and its storage from the remote rpc node into local chain.
```
Comment (ixje) [inline docs/command-reference.md:489]: why is this line in here? It's the same information as in line 469

File: docs/command-reference.md [hunk 2]
```diff
@@ -464,6 +464,31 @@ Arguments:
...
+
+Arguments:
+  Contract              Target contract hash
+  RpcUri                URL of Neo JSON-RPC Node
+                        Specify MainNet (default), TestNet or JSON-RPC URL
+
+[Options]:
+  -i|--input <INPUT>    Path to neo-express data file
+  -h|--height <HEIGHT>  Block height to get contract state for
+                        Default value is: 0.
```
Comment (ixje) [inline docs/command-reference.md:482]: we should explain that default means 'latest' (just like we did in the CLI help text https://github.com/neo-project/neo-express/commit/044200181278f50b2b19e838923904b0bc3b8720)

File: docs/command-reference.md (hunk 1 above)
Comment (ixje) [reply docs/command-reference.md:489]: @Liaojinghui

Comment (Liaojinghui) [reply docs/command-reference.md:489]: #327 Please take a look

**Pull request merged**